frontend/packages/kubevirt-plugin/integration-tests/tests/models/importWizard.ts

```diff
@@ -197,11 +259,32 @@ export class ImportWizard extends Wizard {
     });
   }
 
-  async import(config: VMImportConfig) {
+  async addVmNetworks(networkResources: NetworkResource[]) {
+    // Networking
+    // First update imported network interfaces to comply with k8s
+    await this.updateImportedNICs();
+    // Optionally add new interfaces, if any
+    if (networkResources) {
+      for (const NIC of networkResources) {
+        await this.addNIC(NIC);
+      }
+    }
+  }
+
+  async addVmStorage(storageResources: StorageResource[]) {
+    // Storage
+    // First update disks that come from the source VM
+    await this.updateImportedDisks();
```

Comment:
similarly I wouldn't call this here.
We're in addVmStorage method so the focus should be adding disks.

Reply:
Now calling of `updateImportedDisks` is done separately from import method